src/components/Pill/style.css

```diff
@@ -0,0 +1,51 @@
+@import '../../shared/colors.css';
```

Comment:
PR #69 should have already made colors.css available here, so you can remove this line.